katalog/harbor/trivy/kustomization.yaml

```diff
@@ -17,24 +19,30 @@ commonLabels:
 images:
   - name: goharbor/trivy-adapter-photon
     newName: registry.sighup.io/fury/goharbor/trivy-adapter-photon
-    newTag: v2.4.2
+    newTag: v2.7.0
+
+  - name: registry.sighup.io/fury/kubectl
+    newTag: v1.22.0
 
 configMapGenerator:
   - name: trivy
     literals:
       - HTTP_PROXY=""
       - HTTPS_PROXY=""
-      - NO_PROXY="core,jobservice,database,chartmuseum,trivy,notary-server,notary-signer,registry,portal,127.0.0.1,localhost,.local,.internal"
+      - NO_PROXY="core,jobservice,database,chartmuseum,trivy,notary-server,notary-signer,registry,portal,exporter,127.0.0.1,localhost,.local,.internal"
       - SCANNER_LOG_LEVEL="info"
       - SCANNER_TRIVY_CACHE_DIR="/home/scanner/.cache/trivy"
       - SCANNER_TRIVY_REPORTS_DIR="/home/scanner/.cache/reports"
       - SCANNER_TRIVY_DEBUG_MODE="false"
       - SCANNER_TRIVY_VULN_TYPE="os,library"
       - SCANNER_TRIVY_SEVERITY="UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL"
       - SCANNER_TRIVY_IGNORE_UNFIXED="false"
-      - SCANNER_TRIVY_SKIP_UPDATE="false"
+      - SCANNER_TRIVY_SKIP_UPDATE="true"
```

Comment:
Maybe we should add also the skip for the java db, and use the same approach.

See: https://github.com/sighupio/trivy-offline/pull/13

Reply:
Sorry @ralgozino, I double checked the branch and the indicated parameter is set to "true". Should it be set to false?

Reply:
I did not mean to change `SCANNER_TRIVY_SKIP_UPDATE`.

Trivy has also a DB for Java vulnerabilities. Our trivy-offline image has added support for having that DB cached in the same way we cache the CVEs in the PR that I linked.

I was suggesting checking if we should consider also the Java DB and add the corresponding env var and logic

Reply:
Hi @ralgozino , sorry I didn't understand the previous message. I see the value "true" highlighted and I thought you were referring to that.
I checked the trivy page that uses harbor https://github.com/aquasecurity/harbor-scanner-trivy, but there doesn't seem to be any variables that I can use to disable Java db download. From what is indicated on the link above, only the set of the SCANNER_TRIVY_SKIP_UPDATE="true" variable should be enough to disable downloads.